Lib/fontTools/varLib/__init__.py

```diff
@@ -208,6 +212,62 @@ def _add_avar(font, axes):
         assert +1.0 not in curve or curve[+1.0] == +1.0
         # curve.update({-1.0: -1.0, 0.0: 0.0, 1.0: 1.0})
 
+    if mappings:
+        interesting = True
+
+        hiddenAxes = [axis for axis in axes.values() if axis.hidden]
+
+        inputLocations = [
+            {
+                axes[name].tag: models.normalizeValue(v, vals_triples[axes[name].tag])
+                for name, v in mapping.inputLocation.items()
+            }
+            for mapping in mappings
+        ]
+        outputLocations = [
+            {
+                axes[name].tag: models.normalizeValue(v, vals_triples[axes[name].tag])
+                for name, v in mapping.outputLocation.items()
+            }
+            for mapping in mappings
+        ]
+
+        # If base-master is missing, insert it at zero location.
+        if not any(all(v == 0 for k, v in loc.items()) for loc in inputLocations):
+            inputLocations.insert(0, {})
+            outputLocations.insert(0, {})
+
+        model = models.VariationModel(inputLocations, axisTags)
+        storeBuilder = varStore.OnlineVarStoreBuilder(axisTags)
+        storeBuilder.setModel(model)
+        varIdxes = {}
+        for t in axisTags:
```

Comment:
let's call this `t` just tag for easier reading

Reply:
i'm ok to use single-letter variables on a one-liner generator expression or list comprehension, but here it's used on multiple lines and need to remind myself what it means